plugins/Makefile.am

```diff
@@ -72,20 +70,21 @@ if ENABLE_PLUGINS
 
 # The NCZarr codec libraries
 lib__nczstdfilters_la_SOURCES = NCZstdfilters.c
+lib__nczstdfilters_la_LDFLAGS = ${INSTALLFLAGS}
 lib__nczhdf5filters_la_SOURCES = NCZhdf5filters.c
-
+lib__nczhdf5filters_la_LDFLAGS = ${INSTALLFLAGS}
```

Comment:
You seem to have commented out the `INSTALLFLAGS` assignment on line 20; should the `_LDFLAGS` lines with just that be deleted (allowing the default `AM_LDFLAGS` to serve those plugins, or the `INSTALLFLAGS` assignment uncommented?

Reply:
Uncommented the `INSTALLFLAGS` definition and started it with `AM_LDFLAGS`, that'll help Cygwin builds (and probably Windows).

Doesn't look like there's a "resolve" button like I've seen on some of these.